### DISCUSSION TITLE
Performance of many editors

### DISCUSSION THREAD

**Discussion by jloganolson** [ueberdosis/tiptap#1478] [General Questions]
I've got a form with 10+ tiptap editors on it and I suspect that's going to continue to grow to 30+ (one per input). On that note, when a tiptap editor isn't being edited/updated, is it comparable in performance to a just a regular div of text? Or would it be worth hiding (via vue's v-if or v-show) the tiptap component and just showing a non interactive div until it's clicked/selected?

(I thought I should ask in case it's already known before I take the time to instrument/test!)

**Comment by tmjoen** [comment]
Please update us when you instrument and test — I'm curious about this too.

**Comment by andreasvirkus** [comment]
We're seeing huge initial rendering times already with ~30+ empty editors (with only the base kit extension) - roughly 8+ seconds, so for now we've gone with the regular div approach until the user needs to interact with the editor.
